Demo/Measures-cms/SupplementalDataElements-3.4.000.g.cs

```diff
@@ -153,42 +153,43 @@ Tuples.Tuple_DMgHTLENEHBHWJISQgKZGZVMB f_(Extension E)
 			};
 			bool? v_(Extension @this)
 			{
-				var ak_ = context.Operators.Equal(@this?.Url, "detailed");
+				var aj_ = context.Operators.Equal(@this?.Url, "detailed");
 
-				return ak_;
+				return aj_;
 			};
 			var w_ = context.Operators.WhereOrNull<Extension>(((E is Element)
 					? ((E as Element).Extension)
 					: null), v_);
 			var y_ = context.Operators.SelectOrNull<Extension, DataType>(w_, p_);
 			CqlCode z_(DataType @this)
 			{
-				var al_ = context.Operators.Convert<Coding>(@this);
-				var am_ = FHIRHelpers_4_3_000.ToCode(al_);
+				var ak_ = context.Operators.Convert<Coding>(@this);
+				var al_ = FHIRHelpers_4_3_000.ToCode(ak_);
 
-				return am_;
+				return al_;
 			};
 			var aa_ = context.Operators.SelectOrNull<DataType, CqlCode>(y_, z_);
 			var ab_ = context.Operators.ValueSetUnion((u_ as IEnumerable<CqlCode>), aa_);
 			bool? ac_(Extension @this)
 			{
-				var an_ = context.Operators.Equal(@this?.Url, "text");
+				var am_ = context.Operators.Equal(@this?.Url, "text");
 
-				return an_;
+				return am_;
 			};
 			var ad_ = context.Operators.WhereOrNull<Extension>(((E is Element)
 					? ((E as Element).Extension)
 					: null), ac_);
-			var af_ = context.Operators.SelectOrNull<Extension, DataType>(ad_, p_);
-			var ag_ = context.Operators.SingleOrNull<DataType>(af_);
-			var ah_ = context.Operators.Convert<string>(ag_);
-			var ai_ = new Tuples.Tuple_DMgHTLENEHBHWJISQgKZGZVMB
+			object ae_(Extension @this) => 
+				@this?.Value;
+			var af_ = context.Operators.SelectOrNull<Extension, object>(ad_, ae_);
+			var ag_ = context.Operators.SingleOrNull<object>(af_);
+			var ah_ = new Tuples.Tuple_DMgHTLENEHBHWJISQgKZGZVMB
 			{
 				codes = ab_,
-				display = ah_,
+				display = (ag_ as string),
```

Comment:
This requires scrutiny: a DataType has become an object, then in the old code the datatype is converted using Convert to a string, then assigned to display. In the new code, this object is just cast to a string, without call to Convert - which is almost certainly wrong, unless the SelectOrNull converts this - but the lambda ae_ does not seem to do that.

Reply:
Added to #292

Reply:
Fixed in #308 
![image](https://github.com/FirelyTeam/firely-cql-sdk/assets/4639799/96cf69be-5860-473d-91ca-dd9045d92077)